Google Exercise /index.html

```diff
@@ -50,20 +50,24 @@
             </div>
         </nav>
 
-        <nav>
+        <nav class="footer"> 
             <footer>
-                <div>
-                    Canada
+                <div class="location">
+                    <a href="#"> Canada</a>
                 </div>
-                <div>
-                    Advertising
-                    Business
-                    How Search Works
-               </div>
-                    Privacy
-                    terms 
-               <div>
-
+                <div class="bar1">
+                    <span>
+                        <a href="#"> Advertising</a>
+                        <a href="#">Business</a>
+                        <a href="#">How Search Works</a>
+                    </span>
+             
+                    
+                    <span>
+                        <a href="#">Privacy</a>
+                        <a href="#">Terms</a>
+                        <a href="#">Settings</a>
+                    </span>
                </div>
             </footer>
         </nav>
```

Comment:
You placed your whole content into `<header>` tag and divided it between `<nav>` tags.
Which means that you page is only header and navigation 😅.
It's better to put elements in containers that are related to them.
Navigation goes to `<header>`
Content goes to `<main>`
Footer goes to `<footer>`
Be careful and use proper semantic blocks.

Reply:
Other than that, Google Page is nicely done!
Good job Sahib!